### PR TITLE
swap granite for mistral cleanup + model and recipe housekeeping

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
Should close out #311 
/cc @containers/ai-lab-recipes-maintainers

### CODE REVIEW

**Comment by rhatdan** [comment]
Needs a rebase.

**Comment by Gregory-Pereira** [comment]
Unblocked post rebase, waiting for tests.

**Comment by rhatdan** [comment]
LGTM
Merge when tests ready.

File: recipes/natural_language_processing/rag/README.md [hunk 1]
```diff
@@ -52,18 +52,21 @@ In order to build this application we will need two models, a Vector Database, a
...
 ```bash
+<<<<<<< HEAD
 cd models
 wget https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.2-GGUF/resolve/main/mistral-7b-instruct-v0.2.Q4_K_M.gguf
 cd ../
+=======
+cd ../../../models
+curl -sLO https://huggingface.co/instructlab/granite-7b-lab-GGUF/resolve/main/granite-7b-lab-Q4_K_M.gguf
+cd ../recipes/natural_language_processing/rag
+>>>>>>> 4f8f404 (swapping standard mistral with granite as default)
```
Comment (MichaelClifford) [inline recipes/natural_language_processing/rag/README.md]: This looks like it needs to be resolved.

Comment (Gregory-Pereira) [reply recipes/natural_language_processing/rag/README.md]: Great catch thank you, I will clean this up shortly

**Comment by Gregory-Pereira** [comment]
Unblocked @MichaelClifford

**Review by MichaelClifford — APPROVED** [review]
lgtm

**Pull request merged**